ui/components/keywords.jsx

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+import React from 'react';
+import { resolve } from 'react-resolver';
+import { Link } from 'react-router';
+
+function Keyword(props) {
+  return <li>{props.keyword}</li>;
+}
+
+function Keywords(props) {
+  return (
+    <div>
+      <h1>Keywords</h1>
+      <ul>
+        {props.data.map(keyword => <Link to={{ pathname: '/requirements/', query: { keywords__name__in: keyword.name } }}><Keyword keyword={keyword.name} /></Link>)}
```

Comment:
Something for the future -- we can use `keywords__name=myKeyword` when we're looking for a single match. Those filters are really powerful; I think @donjo may be able to get some interesting data just by manipulating the URL.